CHANGES.md

```diff
@@ -4,6 +4,7 @@
 
 * Support exactly-once consistency in BigQuery sink. This is integrated with Datastream and Table/SQL API.
 * Add Flink metrics for monitoring BigQuery sink.
+* Package unshaded guava dependency for enforcing the correct version used by BigQuery client.
```

Comment:
These changes are already merged, please merge `dataproc/main` into your branch (`shashambhavi/unbound-integration-test`).

Reply:
Thanks for pointing it out! Merged the changes and resolved the conflicts.